MdeModulePkg/Library/PeiDxeDebugLibReportStatusCode/DebugLib.c

```diff
@@ -495,6 +496,12 @@ DebugAssert (
   UINTN                  FileNameSize;
   UINTN                  DescriptionSize;
 
+  // MU_CHANGE BEGIN LOGTELEMETRY
+  UINTN                  FileNameLength = AsciiStrLen (FileName) - 2; // We don't care about the extension

```

Comment:
How do you know to subtract 2?
Can you make this a constant so it's obvious what you're doing?